tests/unit/test_helper_security.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Unit test for the helper_cluster library."""
+import math
+import re
+import unittest
+from datetime import datetime
+
+import ops
+from charms.opensearch.v0.helper_security import (
+    cert_expiration_remaining_hours,
+    generate_hashed_password,
+    generate_password,
+    normalized_tls_subject,
+    rfc2253_tls_subject,
+)
+
+ops.testing.SIMULATE_CAN_CONNECT = True
```

Comment:
since you are using this setup often, consider moving it to `tests/unit/__init__.py`

```python
import ops.testing

ops.testing.SIMULATE_CAN_CONNECT = True
```

Reply:
Thanks Paulo, good idea